### PR TITLE
add creative methodology and update lifecycle docs

### PR DESCRIPTION
**Pull request by bokelley**

### CODE REVIEW

File: docs/creative.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+### Vendor Emissions
+
+For each vendor that participates in delivering and tracking the creative, we need to calculate the emissions per creative delivery request. This is part of the [ad tech vendor methodology](./ad_tech_model.md).
+
+## Input parameters
+
+| Parameter    | Values                      | Required | Comments                                                                    |
+| ------------ | --------------------------- | -------- | --------------------------------------------------------------------------- |
+| Payload Size | bytes                       | yes      | Bytes transferred to fully display the creative                             |
+| Duration     | seconds                     | yes      | Used for consumer device calculation. For banners/native this should be 1.  |
```
Comment (MikeFreyberger) [inline docs/creative.md:38]: should be 1 or has to be 1? is 1 the minimum? is this an integer or a float?

File: docs/lifecycle.md [hunk 2]
```diff
@@ -2,52 +2,47 @@
...
 
-1. Producing the creative
-2. Producing the content
-3. Selecting which ad to display
-4. Consumption of the media and creative, including disposal
-5. Consumer behavior after consuming the ad
+1. Media distribution
+1. Ad selection
+1. Consumer engagement - the impression
+1. Disposal
```
Comment (MikeFreyberger) [inline docs/lifecycle.md:14]: and in parens `(Ad disposal and media disposal)`? just to be very clear the connection between this list and image below.

also, would be cool if the image could somehow have something to mark the sections connected to use and the sections connected to production

File: docs/lifecycle.md [hunk 3]
```diff
@@ -70,35 +65,29 @@ The GHG Protocol provides two methods for tracking Scope 2 emissions: the market
...
-| Includes emissions from programmatic vendors | No | No | Yes | Yes
-| Includes emissions from data and identity providers | No | No | No | Yes
-| Includes emissions from technology embedded into creatives | No | No | Yes | Yes
+DIMPACT is omitted as its methodology does not currently include ad selection.
+
+| Topic                                                      | SRI | GroupM |
+| ---------------------------------------------------------- | --- | ------ |
+| Includes emissions from programmatic vendors               | No  | Yes    |
+| Includes emissions from data and identity providers        | No  | No     |
+| Includes emissions from technology embedded into creatives | No  | Yes?   |
```
Comment (MikeFreyberger) [inline docs/lifecycle.md:82]: not clear what `Yes?` means. a small comment indicating why this isn't clear based on their report would help.

File: docs/creative.md (hunk 1 above)
Comment (pdobrowsky) [inline docs/creative.md:38]: Does having this be 1 reflect an expectation that a consumer will view the ad for 1 second in a display context? For display video should a user also use 1 as the duration? I think the duration has a lot of meaning in devoted ad experiences (streaming, insta story ads, audio) but in a display channel I think this is complicated by the ads and content being shown together.

Comment (bokelley) [reply docs/creative.md:38]: I'm ok with > 1 - thinking something like an interstitial - but I think 1 should be the default. Float is ok. I don't think we should enforce a minimum here but that might change as the industry sets standards.

Comment (bokelley) [reply docs/creative.md:38]: For display video, I think this should be the duration of the video. Arguably an out stream video will not have the full screen worth of emissions for the whole length, but that would be the conservative approach.

File: docs/lifecycle.md (hunk 3 above)
Comment (bokelley) [reply docs/lifecycle.md:82]: will change to "unspecified"

**Review by MikeFreyberger — APPROVED** [review]

**Pull request merged**